### DISCUSSION TITLE
Have the input As-Simple-As-Possible

### DISCUSSION THREAD

**Discussion by zvr** [spdx/spec-parser#8] [Ideas]
Let's ask for the input to be as simple as possible!

It is currently full markdown, because the files also get viewed. If we take to generate nice-looking output in different formats (including Markdown, maybe), then we can ask for simpler input formats.

One particular example: do not allow tables! They are more complicated to parse, so why have them? We can read a list with colon-delimited fields.

I mean instead of:
```Markdown
| Attribute | Value |
| --------- | ----- |
|Name|Document|
|SubclassOf|Element|
|Status|stable|
```

we can ask for:
```Markdown
Attribute: Value
Name: Document
SubclassOf: Element
Status: stable
```

We can parse the latter form much more easily, and if needed we can generate the former format.

What do you all think?

**Comment by nirmal-suthar** [comment]
Just had doubt and point in my mind.
- Can there be any case (in future maybe?) where number of columns will be greater than 2, if so how would those be represented ?
```
Attribute: Field1: Field2
```
- If the name of header field (`Attribute` and `Value` here) are fixed, then I think (correct me if wrong) both the former and latter case are relatively similar to handled with former having extra information to separate header. (Basically the limiter is `|` in md tables)

**Reply by zvr** [reply]
Again, it's up to us what the tables will contain --  we have to finalize all this before Tuesday! I'll work on transforming different actual inputs (from the current spec and the PRs) to a simpler form.

For example, it it's always Attribute: Value, we can ask for `## AV-Table` 

Our processing can definitely handle the first line as a heading and generate the correct Markdown.
